corehq/apps/hqadmin/tests/test_stale_data_in_es.py

```diff
@@ -114,7 +116,11 @@ def _submit_form(self, domain, new_cases=0, update_cases=()):
             for case in update_cases
         ]
 
-        form_xml = FormSubmissionBuilder(form_id=str(uuid.uuid4()), case_blocks=case_blocks).as_xml_string()
+        form_xml = FormSubmissionBuilder(
+            form_id=str(uuid.uuid4()),
+            case_blocks=case_blocks,
+            metadata=TestFormMetadata(xmlns=self.form_xmlns) if self.form_xmlns else None
```

Comment:
It's supposed to be falsy for `TestStaleDataInESSQL` (meaning "use some default xmlns for the forms you submit") and to be 'COUCH_XMLNS_NOT_SUPPORTED' for 'TestStaleDataInESCouch` (meaning "instead of a normal xmlns use the string 'COUCH_XMLNS_NOT_SUPPORTED'"). This is because that's what the couch implementation of `stale_data_in_es` always puts as the xmlns, since it's unable to pull it efficiently.

This is the same situation as case_type for couch cases by the way: they're always set to "COUCH_TYPE_NOT_SUPPORTED" in the couch implementation.